tests/solvers/test_radau5.py

```diff
@@ -794,19 +803,78 @@ def f(t, y):
         y0 = N.array([1.])
         prob = Explicit_Problem(f, y0)
         sim = Radau5ODE(prob)
-        sim.solver = 'f'
+        sim.implementation = 'f'
+
+        err_msg = f'Repeated unexpected step rejections.'
+        with nose.tools.assert_raises_regex(Radau5Error, err_msg):
+            sim.simulate(1.)
+
+    @testattr(stddist = True)
+    def test_sparse_solver_attribute(self):
+        """
+        This tests the error when trying to simulate using the Fortran based solver with sparse linear solver setting enabled
+        """
+        sim = Radau5ODE(self.mod)
+        sim.implementation = 'f'
+        sim.linear_solver = 'SPARSE'
 
-        nose.tools.assert_raises(Radau5Error, sim.simulate, 1.)
+        err_msg = f"Sparse Linear solver not supported for Fortran based implementation, instead use 'implementation' = 'c' or 'linear_solver' = 'DENSE'."
```

Comment:
You are using an f-string here but not formatting the string, I am just adding the comment here in case you missed something.